react-native-gutenberg-bridge/android/build.gradle

```diff
@@ -84,7 +74,9 @@ dependencies {
     } else {
         implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-svg', readHashedVersion('../../package.json', 'react-native-svg', 'dependencies')))
         implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-aztec', submoduleGitHash('../../', 'react-native-aztec')))
-        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-recyclerview-list', 'a2e8ca550412504c32218fd473c55f696f18f1f7'))
+        implementation (waitJitpack('com.github.wordpress-mobile', 'react-native-recyclerview-list', '7286f7494a4fc8bdae987ecbf56265c508caf3c9'))
```

Comment:
Similarly to the SVG package, I merged [the react-native-recyclerview-list PR](https://github.com/wordpress-mobile/react-native-recyclerview-list/pull/13) some moments ago so, friendly reminder to update the hash here too. 🙇‍♂️

BTW, this package is not actually been used in the Gutenberg-mobile app since a few weeks back, but we still haven't completely removed it from the build but, it's good that we keep it up-to-date with this build process changes.